src/internal.rs

```diff
@@ -975,6 +995,6 @@ pub struct Dav1dCdefDSPContext {
 #[derive(Copy, Clone)]
 #[repr(C)]
 pub struct Dav1dLoopRestorationDSPContext {
-    pub wiener: [looprestorationfilter_fn; 2],
-    pub sgr: [looprestorationfilter_fn; 3],
+    pub wiener: [LoopRestorationFilterFn; 2],
+    pub sgr: [LoopRestorationFilterFn; 3],
 }
```

Comment:
This is originally defined in `src/looprestoration.h`, so can you put it in `src/looprestoration.rs`?